ramls/inventory/illpolicy.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "An ILL policy",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "unique ID of the ILL policy; UUID"
+    },
+    "name": {
+      "type": "string",
+      "description": "name of the policy"
+    },
+    "source": {
+      "type": "string",
+      "description": "label indicating where the ILL policy entry originates from, i.e. 'folio' or 'local'"
+    },
+    "metadata": {
+      "type": "object",
+      "$ref": "../raml-util/schemas/metadata.schema",
+      "readonly": true
+    }
+  },
+  "additionalProperties": true,
+  "required": [
+    "name",
+    "source"
```

Comment:
You have made "source" required for this schema and not required for others, like  classificationtype.json or contributornametype.json . Is that OK?